src/components/Navbar.tsx

```diff
@@ -132,42 +165,56 @@ export default function Navbar() {
             preview={false}
           />
         </Link>
-        <Menu
-          onClick={(e) => {
-            setselectedKeys([e.key]);
+
+        {/* Hamburger button for mobile */}
+        <Button
+          className="md:hidden"
+          type="text"
+          icon={<MenuOutlined />}
+          onClick={toggleDrawer}
+          style={{
+            fontSize: "24px",
+            marginLeft: "auto",
+            marginRight: "10px",
+            display: "flex",
+            alignItems: "center",
+            justifyContent: "center",
           }}
+        />
+        {/* New Mobile Menu drawer*/}
+        <Drawer
+          title="Menu"
+          placement="right"
+          onClose={toggleDrawer}
+          open={isDrawerOpen}
+          width={250}
+        >
+          <Menu
+            onClick={(e) => {
+              setselectedKeys([e.key]);
+              toggleDrawer(); // Close drawer on menu item click
+            }}
+            selectedKeys={selectedKeys}
+            theme="dark"
+            mode="vertical"
+            items={menuItems}
+          />
+        </Drawer>
+
+        {/* Normal menu (destop)*/}
+        <Menu
+          onClick={(e) => setselectedKeys([e.key])}
           selectedKeys={selectedKeys}
           theme="dark"
           mode="horizontal"
+          className="hidden md:flex"
```

Comment:
![image](https://github.com/user-attachments/assets/7622e62b-0d9f-4722-a79a-f3637756838a)

So what to do if that doesn't work 😂 See what's next to the logo.